common/persistence/sql/sqlplugin/sqlite/db.go

```diff
@@ -29,6 +29,11 @@ import (
 
 	"github.com/uber/cadence/common/persistence/sql/sqlplugin"
 	"github.com/uber/cadence/common/persistence/sql/sqlplugin/mysql"
+
+	// import sqlite driver
+	_ "github.com/ncruces/go-sqlite3/driver"
+	// import embed sqlite db
+	_ "github.com/ncruces/go-sqlite3/embed"
```

Comment:
Why import it if it is not used? Is it running some init function?

Reply:
Yeah, usually all driver that support `database/sql` should be loaded to run[ their init function](https://github.com/ncruces/go-sqlite3/blob/main/driver/driver.go#L98) to be registered to `database/sql`